pom.xml

```diff
@@ -70,8 +70,8 @@
 
     <osgi-version>4.2.0</osgi-version>
 
-    <scala-version>2.11.0</scala-version>
     <scala-version-suffix>2.11</scala-version-suffix>
+    <scala-version>${scala-version-suffix}.0</scala-version>
```

Comment:
not really, it's just how I prefer it to be (less changes when scala version changed).